src/wizard/ionic/index.md

```diff
@@ -0,0 +1,217 @@
+---
+name: Capacitor
+doc_link: https://docs.sentry.io/platforms/javascript/guides/capacitor/
+support_level: production
+type: framework
+---
+
+Install the Sentry Capacitor SDK alongside the sibling Sentry SDK:
+
+```angular {tabTitle: Ionic + Angular}
+# npm
+npm install --save @sentry/capacitor @sentry/angular
+
+# yarn
+yarn add @sentry/capacitor @sentry/angular @sentry/tracing
+```
+
+```react {tabTitle: Ionic + React}
+# npm
+npm install --save @sentry/capacitor @sentry/react 
+
+# yarn
+yarn add @sentry/capacitor @sentry/react  @sentry/tracing
+```
+
+```vue {tabTitle: Ionic + Vue}
+# npm
+npm install --save @sentry/capacitor @sentry/vue 
+
+# yarn
+yarn add @sentry/capacitor @sentry/vue  @sentry/tracing
+```
+
+## Android Installation
+
+Then, add the `SentryCapacitor` plugin class inside the `onCreate` method of your `MainActivity` file.
+
+Java:
+
+```java
+import android.os.Bundle;
+import com.getcapacitor.BridgeActivity;
+import com.getcapacitor.Plugin;
+import io.sentry.capacitor.SentryCapacitor;
+import java.util.ArrayList;
+
+public class MainActivity extends BridgeActivity {
+  @Override
+  public void onCreate(Bundle savedInstanceState) {
+    super.onCreate(savedInstanceState);
+    // Initializes the Bridge
+    this.init(savedInstanceState, new ArrayList<Class<? extends Plugin>>() {{
+      add(SentryCapacitor.class);
+    }});
+  }
+}
+```
+
+Kotlin:
+
+```kotlin
+import android.os.Bundle
+import com.getcapacitor.BridgeActivity
+import com.getcapacitor.Plugin
+import io.sentry.capacitor.SentryCapacitor
+
+class MainActivity : BridgeActivity() {
+  fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    // Initializes the Bridge
+    this.init(
+      savedInstanceState,
+      listOf<Class<out Plugin>>(SentryCapacitor::class.java)
+    )
+  }
+}
+```
+
+## Initializing the SDK
+
+You must initialize the Sentry SDK as early as you can:
+
+```angular {tabTitle: Angular}
+// app.module.ts
+import * as Sentry from "@sentry/capacitor";
+import * as SentryAngular from "@sentry/angular";
+// If taking advantage of automatic instrumentation (highly recommended)
+import { BrowserTracing } from "@sentry/tracing";
+// Or, if only manually tracing
+// import "@sentry/tracing";
+// Note: You MUST import the package in some way for tracing to work
```

Comment:
Should we just add to the example and remove these comments? Just add the import and the example (e.g: Go with performance)

Reply:
I just removed the comments since by default it'll automatically create transactions when loading a page.